src/mudlet.cpp

```diff
@@ -2059,30 +2087,46 @@ void mudlet::closeEvent(QCloseEvent* event)
         }
     }
 
+    // pass the event on so dblsqd can perform an update
+    // if automatic updates have been disabled
     event->accept();
-    qApp->quit();
```

Comment:
Yeah. It also prevents the updater from firing off the "last window is closed" to show the update dialog because of the explicit and redundant quit call.

Reply:
OK.